File: JS Exams/01.  Basics Exam Collection/Basics Online Retake Exam - 2 and 3 May 2019/06. Vet Parking/06. Vet Parking.js
function vetParking(arg) {
const [days, hours] = [...arg.map(Number)]

    let currentTax = 0
    let totalTax = 0
    for (let day = 1; day <= days; day++) {
        for (let hour = 1; hour <= hours; hour++) {
            if ((day & 1) == 0 && hour & 1) {
                currentTax += 2.50
                totalTax += 2.50
            } else if (day & 1 && (hour & 1) == 0) {
                currentTax += 1.25
                totalTax += 1.25
            } else {
                currentTax++
                totalTax++
            }
        }

        console.log(`Day: ${day} - ${currentTax.toFixed(2)} leva`)
        currentTax = 0
    }

    console.log(`Total: ${totalTax.toFixed(2)} leva`)
}

//vetParking([2, 5])

File: Python Exams/01. Basic Exam Collection/Basics Online Retake Exam - 2 and 3 May 2019/06. Vet Parking/06. Vet Parking.py
days = int(input())
hours = int(input())
current_tax = 0
total_tax = 0

for day in range(1, days + 1):
    for hour in range(1, hours + 1):
        if not day & 1 and hour & 1:
            current_tax += 2.50
            total_tax += 2.50
        elif day & 1 and not hour & 1:
            current_tax += 1.25
            total_tax += 1.25
        else:
            current_tax += 1
            total_tax += 1

    print(f'Day: {day} - {current_tax:.2f} leva')
    current_tax = 0


print(f'Total: {total_tax:.2f} leva')

